spinup_bis/algos/tf2/sop/sop.py

```diff
@@ -467,14 +412,12 @@ def reset_episode(epoch):
                                LossPi=results['pi_loss'],
                                LossQ1=results['q1_loss'],
                                LossQ2=results['q2_loss'])
-                for idx, (q1, q2) in enumerate(
-                        zip(results['q1'], results['q2'])):
-                    metrics.update({
-                        f'Q1Vals_{idx + 1}': q1,
-                        f'Q2Vals_{idx + 1}': q2,
-                        f'QDiff_{idx + 1}': np.abs(q1 - q2),
-                    })
-                logger.store(**metrics)
+                metrics.update({
+                    f'Q1Vals': results['q1'],
+                    f'Q2Vals': results['q2'],
+                    f'QDiff': np.abs(results['q1'] - results['q2']),
+                })
+            logger.store(**metrics)
```

Comment:
Shouldn't it be indented and aligned with `metrics.update`?

Reply:
done